pkgs/by-name/ch/chrome-remote-desktop/package.nix

```
@@ -0,0 +1,114 @@
+{ stdenvNoCC
+, lib
+, autoPatchelfHook
+, dpkg
+, fetchurl
+, glib
+, gtk3
+, libdrm
+, libutempter
+, mesa
+, nss
+, pam
+, python3
+, shadow
+, xorg
+}:
+
+let
+  replacePrefix = "/opt/google/chrome-remote-desktop";
+in
+stdenvNoCC.mkDerivation rec {
+  name = "chrome-remote-desktop";
```

```suggestion
  pname = "chrome-remote-desktop";
```

or any special reason to use `name` here?